dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/ThumbnailServiceWrapper.java

```
@@ -191,132 +140,48 @@ public String getDatasetCardImageAsBase64Url(Dataset dataset, Long versionId, bo
-                }
-            }
-
-            if (dataFileService.isThumbnailAvailable(thumbnailImageFile)) {
-                cardImageUrl = ImageThumbConverter.getImageThumbnailAsBase64(
-                        thumbnailImageFile,
-                        ImageThumbConverter.DEFAULT_CARDIMAGE_SIZE);
-            }
-
-            if (cardImageUrl != null) {
-                this.dvobjectThumbnailsMap.put(thumbnailImageFileId, cardImageUrl);
-            } else {
-                this.dvobjectThumbnailsMap.put(thumbnailImageFileId, "");
-            }
-        }
-
-        //logger.info("dataset id " + result.getEntityId() + ", returning " + cardImageUrl);
-
-        return cardImageUrl;
+        this.dvobjectThumbnailsMap.put(datasetId, null);
+        return null;
     }
 
     // it's the responsibility of the user - to make sure the search result
     // passed to this method is of the Dataverse type!
     public String getDataverseCardImageAsBase64Url(SolrSearchResult result) {
-        return dataverseService.getDataverseLogoThumbnailAsBase64ById(result.getEntityId());
+        Thumbnail thumbnail = dataverseThumbnailService.getDataverseLogoThumbnail(result.getEntityId());
+        return ThumbnailUtil.thumbnailAsBase64(thumbnail);
     }
 
     public void resetObjectMaps() {
         dvobjectThumbnailsMap = new HashMap<>();
         dvobjectViewMap = new HashMap<>();
     }
 
+    private void updateDataFileFromSearchResult(SolrSearchResult result) {
+        if (result.getTabularDataTags() != null) {
+            for (String tabularTagLabel : result.getTabularDataTags()) {
+                DataFileTag tag = new DataFileTag();
+                try {
+                    tag.setTypeByLabel(tabularTagLabel);
+                    tag.setDataFile((DataFile) result.getEntity());
+                    ((DataFile) result.getEntity()).addTag(tag);
+                } catch (IllegalArgumentException iax) {
+                    // ignore 
```

Isn't ignoring exceptions anti-pattern :P?